orchestrator/api/documents_v2.py

```
@@ -0,0 +1,1310 @@
+        # Check for duplicate
+        existing = db.query(Document).filter(Document.content_hash == content_hash).first()
+        if existing:
+            return DocumentUploadResponse(
+                document_id=existing.id,
+                filename=existing.filename,
+                status="duplicate",
+                message="Document already exists"
+            )
+        
+        # Save file temporarily
+        upload_dir = Path("/tmp/automotas_uploads")
```

Ruff (0.13.2)

Probable insecure usage of temporary file or directory: "/tmp/automotas_uploads"

(S108)